packages/fast-components-styles-msft/src/call-to-action/index.ts

```diff
@@ -91,10 +93,8 @@ const styles: ComponentStyles<CallToActionClassNameContract, DesignSystem> = (
             width: "8px",
             [applyLocalizedProperty("marginLeft", "marginRight", direction)]: "6px",
             transform: direction === Direction.ltr ? "none" : "rotate(180deg)",
-            verticalAlign: "top",
             transition: "all 600ms cubic-bezier(0.19, 1, 0.22, 1)",
-            left: "0",
-            marginTop: "4px",
+            marginTop: direction === Direction.ltr ? "4px" : "0",
```

Comment:
Why is he `marginTop` value concerned with document direction?

Reply:
RTL is weird.